xtext-android-gradle-plugin/src/main/java/org/xtext/gradle/android/XtextAndroidBuilderPlugin.xtend

```diff
@@ -78,7 +78,9 @@ class XtextAndroidBuilderPlugin implements Plugin<Project> {
 			generatorTask.dependsOn(variant.outputs.map[processResources])
 			variant.javaCompiler.doLast(new Action<Task>() {
 				override void execute(Task it) {
-					generatorTask.installDebugInfo()
+					if (it instanceof AbstractCompile) {
```

Comment:
This is always true, you are configuring a JavaCompile task here.

Reply:
same as above. in the api the task is of type task
=> need instanceof / downcast anway